eve/workers/pod-builder/Dockerfile

```diff
@@ -19,11 +19,32 @@ RUN yum install -y epel-release \
     python36-pip \
     genisoimage \
     git \
+    hg \
     skopeo \
     yum-utils \
     docker-ce-cli-18.09.6 \
     && adduser -u 1042 --home /home/eve eve --groups docker \
-    && chown eve:eve /home/eve/workspace \
+    && chown -R eve:eve /home/eve \
     && pip install buildbot-worker==${BUILDBOT_VERSION}
 
+# Install Go tooling to build the Kubernetes Operators.
+ENV GO_VERSION 1.12.7
+ENV OPERATOR_SDK_VERSION v0.9.0
+ENV GOROOT /usr/local/go
+ENV GOPATH /home/eve/.golang
+ENV GOCACHE /home/eve/.cache
+ENV PATH $GOPATH/bin:$GOROOT/bin:$PATH
+
+RUN curl -ORL https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz      \
+    && tar xzvf go${GO_VERSION}.linux-amd64.tar.gz                             \
+    && rm go${GO_VERSION}.linux-amd64.tar.gz                                   \
+    && mv go /usr/local                                                        \
+    && curl -RLo /usr/bin/operator-sdk                                         \
+        https://github.com/operator-framework/operator-sdk/releases/download/${OPERATOR_SDK_VERSION}/operator-sdk-${OPERATOR_SDK_VERSION}-x86_64-linux-gnu \
+    && chmod +x /usr/bin/operator-sdk
+
 USER eve
+
+# Pre-download the Go dependencies.
+COPY storage-operator/go.mod storage-operator/go.sum ./
```

Comment:
Since you're only `COPY`ing from `storage-operator/`, maybe the build context could be just `'storage-operator/'` instead of `'.'`?

Reply:
Indeed. Done.